app/controllers/hyrax/batch_edits_controller.rb

```diff
@@ -97,15 +116,15 @@ def destroy_batch
     end
 
     def form_class
-      Forms::BatchEditForm
+      Hyrax.config.use_valkyrie? ? Forms::ResourceBatchEditForm : Forms::BatchEditForm
```

Comment:
nitpick: How's Hyrax feel about ternaries over explicit branches? We use an if/else above, should we use one here?

Reply:
if it's not being flagged by rubocop, i think we don't block on style.

on the other hand, i'm wondering if we could just drop all the old handling? if the valkyrie code path is working, do we need the opt-in?